docs/integrating/components.md

```diff
@@ -0,0 +1,65 @@
+# Components
+This document is general advice for creating and working with new components.
+
+## Table of Contents
+
+- [Referencing source files](#referencing-source-files)
+- [Supporting files](#supporting-files)
+- [Connecting a component to state](#connecting-a-component-to-state)
+
+## Referencing source files
+To make referencing source files easier we have set up three simple aliases when working inside an app.
+
+* `@client` which points to `src/client/*`
```

Comment:
These aren't necessary for components right? We should always be aliasing to `@src`

Reply:
This is still true for how we do our apps. I think we could probably make a section that is more recommended practices since it is configurable. Can follow up with better docs later.